src/module/add-like.js

```diff
@@ -0,0 +1,15 @@
+async function addLike(id) {
+    fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/otoY0fxXk5LjLMlhzjv8/likes', {
+      method: 'POST',
+      body: JSON.stringify({
+        item_id: id,
+        likes: 1,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+        Accept: 'application/json',
+      },
+    })
+      .then((response) => response.json());
+  }
+  export default addLike;
```

Comment:
- [ ] when I click on like the first it doesn't respond instantly and may confuse the user so please make sure to fix that it should when I click the number of likes should be incremented 
- I suggest from me to do that try when someone click on like try to get the number of likes from the from  dom and incremented in the dom then try to send it to the API that way it will be fast